docs/release-taxonomy.md

```diff
@@ -0,0 +1,33 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+**Hyperledger Governing Process document - do not modify without first getting approval of the Technical Steering Committee**
+
+Releases at the Hyperledger Project are done according to the [semver.org](https://semver.org) [taxonomy](#1). In addition to the semantic versioning scheme, where we use MAJOR.MINOR.PATCH numbering, following the established guidance given in the semver specification, it is also strongly encouraged that projects should use the following tags, as permitted by semver:
```

Comment:
`the Hyperledger Project` -> `Hyperledger`

Not sure if you are doing a direct port or if changes like this should be suggested.

Reply:
I'm fine with any changes